class ValidateException(BaseException):
    def __init__(self):
        ...
